src/main/java/bean/command/Parser.java

```diff
@@ -0,0 +1,49 @@
+package bean.command;
+
+public class Parser {
+    private final String[] fieldValuePairs;
+    private final String command;
+
+    public String getValue(String field) {
+        for (String item : fieldValuePairs) {
+            if (item.startsWith(field) && item.contains(" ")) {
+                int indexOfFirstSpace = item.indexOf(" ");
+                return item.substring(indexOfFirstSpace + 1).trim();
+            }
+        }
+        return null;
+    }
```

Comment:
Is it possible to do a guard clause for [happy path](https://se-education.org/guides/conventions/java/basic.html) here?